client/extensions/woocommerce/app/orders/orders-filter-nav.js

```diff
@@ -29,16 +29,28 @@ class OrdersFilterNav extends Component {
 
 	render() {
 		const { translate, site, status } = this.props;
+		let currentSelection = translate( 'All orders' );
+		if ( 'pay' === status ) {
+			currentSelection = translate( 'Awaiting Payment' );
+		} else if ( 'fulfill' === status ) {
+			currentSelection = translate( 'Awaiting Fulfillment' );
+		} else if ( 'finished' === status ) {
+			currentSelection = translate( 'Completed' );
+		}
+
 		return (
-			<SectionNav>
-				<NavTabs label={ translate( 'Status' ) } selectedText={ translate( 'All orders' ) }>
+			<SectionNav selectedText={ currentSelection }>
+				<NavTabs label={ translate( 'Status' ) } selectedText={ currentSelection }>
 					<NavItem path={ getLink( '/store/orders/:site', site ) } selected={ 'any' === status }>
 						{ translate( 'All orders' ) }
 					</NavItem>
-					<NavItem path={ getLink( '/store/orders/processing/:site', site ) } selected={ 'processing' === status }>
-						{ translate( 'Processing' ) }
+					<NavItem path={ getLink( '/store/orders/pay/:site', site ) } selected={ 'pay' === status }>
```

Comment:
Would you consider `unpaid` instead of `pay` and `unfulfilled` instead of `fulfill` ?